LICENSE.Apache-2.txt

```diff
@@ -0,0 +1,182 @@
+Applies to pyro/contrib/autoname/compilation.py, which is modified from 
```

Comment:
It's common to create a `vendor/` subdirectory and embed other code there, including license.